liota/core/package_manager.py

```diff
@@ -1140,13 +1162,20 @@ def _package_delete(self, file_name):
     def _terminate_all(self):
         global package_messenger_thread
         global package_messenger_pipe
+        global package_responser_pipe
 
         log.info("Shutting down package messenger...")
         if package_messenger_thread.isAlive():
             with open(package_messenger_pipe, "w") as fp:
                 fp.write(
                     "terminate_messenger_but_you_should_not_do_this_yourself\n")
 
+        log.info("Shutting down package responser...")
+        if package_messenger_thread.isAlive():
+            with open(package_responser_pipe, "w") as fp:
+                fp.write(
+                    "terminate_responser_but_you_should_not_do_this_yourself\n")
```

Comment:
What does this actually do? Same Q for 1171.

Reply:
Thought this over again and feel it is not necessary, so removed it.
The previous purpose is that when execute a "terminate" cmd, notify the user that package thread is executing it. But since we will pass "Success" back, no need to do this.